lang/LangSource/SC_TerminalClient.h

```diff
@@ -55,25 +55,17 @@ class SCLANG_DLLEXPORT SC_TerminalClient : public SC_LanguageClient {
     };
 
     struct Options : public SC_LanguageClient::Options {
-        Options():
-            mLibraryConfigFile(0),
-            mDaemon(false),
-            mCallRun(false),
-            mCallStop(false),
-            mStandalone(false),
-            mArgc(0),
-            mArgv(0) {}
-
-        const char* mLibraryConfigFile;
+        Options(): mDaemon(false), mCallRun(false), mCallStop(false), mStandalone(false) {}
```

Comment:
Just a comment: since C++11 you can initialize member variables inline. In this case, `Options` wouldn't even need a user-defined constructor. In fact, that's what you are already doing in `CLIOptions`. Generally, in modern C++ we only tend to write constructors when they are really necessary.